apps/frontend/src/app/pages/profile/profile-follow/profile-follow.component.ts

```diff
@@ -31,6 +32,7 @@ export class ProfileFollowComponent implements OnInit {
   ngOnInit() {
     this.userSubject.subscribe((usr) => {
       this.user = usr;
+      this.isLoggedInUser = this.user.id === this.localUserService.localUser.id;
```

Comment:
Like Gordi says profile.component has this, just make a new `@Input' in this class and pass it in from the parent via the binding.

Reply:
Using Gordi's approach I think we can just handle showing/hiding this component at the parent component level. Seems we don't need to pass anything in any more.